falguard.py

```diff
@@ -61,10 +63,17 @@ def to_dict(self, obj_type=dict):
 
 
 class Validator:
-
     def __init__(self, spec_path):
+        deserializers = {
+            '.json': json.loads,
+            '.yaml': yaml.load,
+            '.yml': yaml.load
+        }
         with open(spec_path, 'r') as f:
-            spec_dict = json.loads(f.read())
+            # pylint: disable=unused-variable
+            filename, ext = os.path.splitext(spec_path)
```

Comment:
1. Instead of `# pylint: disable=unused-variable` you could just `_, ext = ...`. I have checked and pylint doesn't complain!
2. Also, this line should be outside the context manager.

Reply:
1. Ok nice to know.
2. If i use `_, ext = ... ` expression i don't need to move anything? I may have misunderstood

Reply:
2. Nope, I still want it outside. It is similar to try/except. You want to put only bare minimum inside with/try blocks. It's good for debugging and it's good for readability, the code _will_ change in the future and we should make it easier for next generations :)